### PR TITLE
Getting ready for release 1.5.0

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

**Comment by hardbyte** [comment]
While reviewing https://github.com/data61/python-paillier/pull/99 I noticed python-paillier never migrated off `travis-ci.org` - which is now in `readonly` mode. This PR switches to github actions.

@wilko77 if you have the power can you change the required status checks for this repository?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```
Comment (wilko77) [inline .github/workflows/test.yml:16]: if we claim that our library works with python 3.7 - 3.11, maybe we should test all those versions?

File: README.rst [hunk 2]
```diff
@@ -69,9 +69,9 @@ https://code.google.com/p/encrypted-bigquery-client/
...
     :alt: Latest released version on PyPi
 
-.. |travisM| image:: https://travis-ci.org/data61/python-paillier.svg?branch=master
-    :target: https://travis-ci.org/data61/python-paillier
-    :alt: CI status of Master
+.. |ciTest| image:: https://github.com/data61/python-paillier/actions/workflows/test.yml/badge.svg
+    :target: https://github.com/data61/python-paillier/actions/workflows/test.yml
+    :alt: CI Status
 
 .. |reqM| image:: https://requires.io/github/data61/python-paillier/requirements.svg?branch=master
```
Comment (wilko77) [inline README.rst:76]: the requires.io link doesn't work.

**Review by wilko77 — APPROVED** [review]
sweet

**Pull request merged**